mkosi/config.py

```diff
@@ -1964,6 +1964,11 @@ def load_config(args: argparse.Namespace) -> MkosiConfig:
             output = prefix
         args.output = Path(output)
 
+    if "_" in str(args.output) and args.output_format in [OutputFormat.subvolume]:
+        old = str(args.output)
+        args.output = old.replace("_", "-")
```

Comment:
This would leave `args.output` as string and a few lines below we call `args.output.absolute()`, which would then fail. You maybe want something similar to
```suggestion
        args.output = args.output.with_name(args.output.name.replace("_", "-"))
```

Reply:
Ooops! still getting up to speed with the intricacies of modern Python.